src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -49,19 +49,10 @@ use mz_sql::names::{
     ObjectId, QualifiedItemName, ResolvedDatabaseSpecifier, ResolvedIds, ResolvedItemName,
     SchemaSpecifier, SystemObjectId,
 };
+use mz_sql::plan;
```

Comment:
Do you think this is worth a comment explaining why we import just the module and not all of the individual types?